ansible/roles/jitsi-videobridge/files/configure-jvb-shards-wrapper.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+[ -z "$SHARDS_LOGDIR" ] && SHARDS_LOGDIR="/var/log/jitsi/jvb-shards"
+[ -z "$TEMPLATE_LOGFILE" ] && TEMPLATE_LOGFILE="$TEMPLATE_LOGDIR/jvb-reconfigure.log"
+
+if [ ! -d "$TEMPLATE_LOGDIR" ]; then
+  mkdir $TEMPLATE_LOGDIR
+fi
+
+if [ ! -f "$TEMPLATE_LOGFILE" ]; then
+  touch $TEMPLATE_LOGFILE
+fi
+
+cp /etc/jitsi/videobridge/shards.json /var/log/jitsi/jvb-shards/$(date --utc +%Y-%m-%d_%H:%M:%S.Z)-shards.json
+
+echo "$(date --utc +%Y-%m-%d_%H:%M:%S.Z) starting configure-jvb-shards.sh" >> $TEMPLATE_LOGFILE
+/usr/local/bin/configure-jvb-shards.sh
```

Comment:
Shouldn't we also redirect the stdout here to the logfile?
```suggestion
/usr/local/bin/configure-jvb-shards.sh >> $TEMPLATE_LOGFILE
```